# Battery status: voltage, percent, time, and plugged in indication

Header      header

float32     voltage
float32     watt
float32     percent
float32     temp
bool        plugged_in
